virttest/utils_conn.py

```diff
@@ -1946,3 +1979,181 @@ def conn_setup(self):
                 raise ConnServerRestartError(detail)
 
         logging.debug("UNIX connection setup successfully.")
+
+
+class UNIXSocketConnection(ConnectionBase):
+    """
+    Connection of Unix Socket transport.
+
+    Some specific variables for UNIXSocketConnection class.
+
+    setup_on_local: Whether to setup connection from local
+    desturi_port: desturi port to listen on
+    migrateuri_port: migrateuri port to listen on
+    disks_uri_port: disks-uri port to listen on
+    desturi_sock_listen: desturi socket path to listen on
+    migrateuri_sock: migrateuri socket path to listen on or connect to
+    disks_uri_sock: disks-uri socket path to listen on or connect to
+    desturi_sock_conn: desturi socket path to connect to
+    remove_my_rpcworker_policy: wheather to remove my-rpcworker policy
+    remove_qemu_kvm_policy: wheather to remove qemu-kvm policy
+    pmsocat_path: pmsocat script path
+    remote_pmsocat_path: pmsocat script path on remote
+    """
+    __slots__ = ('setup_on_local', 'desturi_port', 'migrateuri_port',
+                 'disks_uri_port', 'desturi_sock_listen', 'migrateuri_sock',
+                 'disks_uri_sock', 'desturi_sock_conn',
+                 'remove_my_rpcworker_policy', 'remove_qemu_kvm_policy',
+                 'pmsocat_path', 'remote_pmsocat_path')
+
+    def __init__(self, *args, **dargs):
+        """
+        Initialization of TLSConnection.
+        """
```

Comment:
@Yingshun Could you add some sample code?

Reply:
updated.